### DISCUSSION TITLE
My name isn't underlined in the authors list on the publications page

### DISCUSSION THREAD

**Discussion by surakshavinod** [alshedivat/al-folio#2852] [Q&A] [answered]
Hi, as the title says, my name doesn't get underlined in the authors list on the publications page and I'm not sure why this is happening. I don't have a middle name so I've left the middle name blank in the config.yml file

**Comment by george-gca** [comment, marked as answer]
[This](https://github.com/alshedivat/al-folio/blob/9674cfc8e54f7cac7a890e4e54bed001cc897192/_layouts/bib.liquid#L61) is where we check for the author information if you want to understand how this works. Also, have you set your information correctly on the [Jekyll scholar](https://github.com/alshedivat/al-folio/blob/9674cfc8e54f7cac7a890e4e54bed001cc897192/_config.yml#L313) section of your `_config.yml`?

**Comment by surakshavinod** [comment]
I had checked the bib.liquid file but seemed like I made a typo in the _cofig.yml file. Thank you!
